src/PublicShareSidebarTrigger.vue

```diff
@@ -72,5 +73,10 @@ export default {
 	display: flex;
 	justify-content: center;
 	height: 44px !important;
+
+	& .sidebar-button {
+		color: #ffffff;
```

Comment:
Shouldn't be variable-defined?

Reply:
We have a variable for it in stable30+, for 29 I don't see anything useful.
This is a copy of https://github.com/nextcloud/server/pull/51599/commits/af55ec81fd123795bf62ed60579cfe2b48e1a6af